styles/AdvancedEditor.css

```diff
@@ -631,6 +633,7 @@ span.comment-marker {
 
 .keywords > .block-section_title > .block {
   align-items: center;
+  padding:  12px 10px 12px
```

Comment:
this will ruin keyword style 
![Screenshot from 2024-07-15 14-50-47](https://github.com/user-attachments/assets/47d00c01-3f55-45ed-90e1-f22f6ef56c81)

to fix this issue you need to include this [selector](https://github.com/Atypon-OpenSource/manuscripts-body-editor/blob/7514acc6189bfb066602d09d6a10644c56d02aa6/styles/AdvancedEditor.css#L528-L535) also when there is a `selected-suggestion` element

Reply:
I have added the margins into keyword instead

Reply:
you don't need to add a margin to fix issue, just update selector to: 
```
.ProseMirror > .keywords > .block-section_title > h1, .ProseMirror > .selected-suggestion > .keywords > .block-section_title > h1{
.....}
```

Reply:
done. please check